jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTableProperties.java

```diff
@@ -57,9 +58,11 @@ public ExamplesTableProperties(String propertiesAsString, String defaultHeaderSe
 
     private Map<String, String> parseProperties(String propertiesAsString) {
         Map<String, String> result = new LinkedHashMap<>();
-        for (String propertyAsString : StringUtils.split(propertiesAsString, ',')) {
-            String[] property = StringUtils.split(propertyAsString, "=", 2);
-            result.put(property[0].trim(), property[1].trim());
+        if (!isBlank(propertiesAsString)) {
```

Comment:
why is this check required?

Reply:
`StringUtils.split` and `split()` of String work different ways